### PR TITLE
[NUI] update nui-widget-app guide

### PR DESCRIPTION
**Pull request by tscholb**
add case for "Destroy a NUI Widget View"

### Change Description ###

Describe your changes here.

For instance,
 - Fix broken links of native application docs.
 -  ...

### Bugs Fixed ###

Provide links to bugs here

For instance,

 - Issue #265

### API Changes ###

In case of ACR, put the link of ACR

For instance,

 - ACR: ACR-1120

### CODE REVIEW

File: docs/application/dotnet/guides/applications/nui-widget-app.md [hunk 1]
```diff
@@ -236,6 +236,23 @@ You can create a widget manager using two different constructors:
...
+     ...
+     myWidgetView.TerminateWidget();
+     Window.Instance.GetDefaultLayer().Remove(myWidgetView);
+     myWidgetView.Dispose();
+     myWidgetView = null;
+     ...
+   }
+   ```
+
 ## Related Information
```
Comment (nawab-samsung) [inline docs/application/dotnet/guides/applications/nui-widget-app.md:256]:
```suggestion
## Related information

File: docs/application/dotnet/guides/applications/nui-widget-app.md [hunk 2]
```diff
@@ -236,6 +236,23 @@ You can create a widget manager using two different constructors:
          ...
        ```
 
+## Destroy a NUI Widget View
+
+To terminiate a widget view, use `TerminateWidget()`.
+For terminate the widget view, All resources for widget must be released.
```
Comment (nawab-samsung) [inline docs/application/dotnet/guides/applications/nui-widget-app.md:242]:
```suggestion
To terminate the widget view, use `TerminateWidget()`, and also ensure to release all the resources used by the widget view:

**Review by nawab-samsung — CHANGES_REQUESTED** [review]
@tscholb Request you to kindly incorporate the comments.

**Comment by nawab-samsung** [comment]
@tscholb  Request you to kindly incorporate the comments.

**Comment by nawab-samsung** [comment]
@tscholb Gentle Reminder: 
Request you to kindly incorporate the comments.

**Comment by nawab-samsung** [comment]
@tscholb Gentle Reminder:
Request you to kindly incorporate the comments.

**Comment by nawab-samsung** [comment]
@tscholb Gentle Reminder:
Request you to kindly incorporate the comments.

**Comment by nawab-samsung** [comment]
@tscholb Gentle Reminder:
Request you to kindly incorporate the comments.

**Comment by tscholb** [comment]
I'm sorry but this PR need to delete.
I will upload it again after revision.

**Pull request closed without merging**